Assets/Scripts/GameTime/Clock.cs

```diff
@@ -6,6 +6,10 @@ namespace GameTime
 {
     public class Clock : MonoBehaviour
     {
+        public delegate void SunRise();
```

Comment:
I'd suggest just creating one delegate called something like `SunStatus` or `DayTimeEvent` and using this for both events